package.json

```
@@ -149,7 +150,8 @@
     "e2e": "ts-node scripts/e2e.ts",
     "e2e:dev": "testcafe chrome --host localhost --skip-js-errors",
     "prepare": "husky install",
-    "chromatic": "chromatic"
+    "chromatic": "chromatic",
+    "export:ui-props": "yarn build:lib && ts-node scripts/exportUIProps.ts"
```

scripts 内は yarn に依存していないように見えるため、run-s で書き換えた方が良いと考えました。

```suggestion
    "export:ui-props": "run-s build:lib 'ts-node scripts/exportUIProps.ts'"
```

提案の記述動作の確認はしていません:pray:

---

@uknmr ありがとうございます！確かにscripts内には`yarn`は出てきていないですね。

サジェストいただいたコードを手元で検証してみたのですが、

> ERROR: Task not found: "ts-node"

となってしまい、[ドキュメント](https://github.com/mysticatea/npm-run-all/blob/master/docs/run-s.md)など確認したところ`run-s`にはnpmスクリプトしか渡せないようでした。

ですので、例えば以下のように2つのタスクを定義するのはどうでしょうか？これであれば動作は大丈夫でした。
```
"write:ui-props": "ts-node scripts/exportUIProps.ts",
"export:ui-props": "run-s build:lib write:ui-props"
```

---

分けて定義するで LGTM です!

---

ありがとうございます、修正しました：https://github.com/kufu/smarthr-ui/pull/3267/commits/56751396f12e5711e7a282f77589ce9548f25994